contrib/memtest/main.go

```diff
@@ -109,8 +110,14 @@ func memory() {
 			counter++
 		}
 	}
-	fmt.Printf("[%d] Current Memory: %05.2f G. Increase? %v\n",
-		counter, float64(curMem)/float64(1<<30), increase)
+	var js z.MemStats
+	z.ReadMemStats(&js)
```

Comment:
Do you need this in memtest?

Reply:
I was thinking that this can be something which we can use to verify that the stats are as expected.